keps/0009-operator-toolkit.md

```diff
@@ -106,93 +100,80 @@ An operator package is a folder that contains all of the manifests needed to cre
 
 #### operator.yaml
 
-`operator.yaml` is the base definition of an operator. It follows the following format, extracted from the MySQL example:
+`operator.yaml` is the base definition of an operator.  It follows the following reference format.
 
 ```yaml
-name: operator
-description: operator
-version: "5.7"
-kudoVersion: ">= 0.2.0"
-kubeVersion: ">= 1.14"
+name: str(max=253, lowercase alphanumeric characters, -, and .)
+description: str(required=False)
+version: semver(desc='version of operator')
+appVersion: str(desc='component controlled version', required=False)
```

Comment:
I think we could do better with desc here. I am not sure what is component and how is it controlling a version. I feel like it's hard to explain the difference between these two versions in one sentence though :/

Reply:
I think the verbosity of the difference will need to be elaborated in docs